medical_appointment/data/medical_appointment_data.xml

```diff
@@ -1,23 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
 <!--
-    Copyright (C) 2015-TODAY LasLabs [https://laslabs.com]
-    @author Dave Lasley <dave@laslabs.com>
-    @license AGPL-3
+    Copyright 2015-2017 LasLabs Inc.
+    License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 -->
 
 <openerp>
```

Comment:
`<odoo>` tag. Also the two data tags can be removed to combine the sections - they're both noupdate=0